src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -68,25 +73,25 @@ running these commands.
 
 The `--dump-dom` flag prints `document.body.innerHTML` to stdout:
 
-    chrome --headless --dump-dom https://www.chromestatus.com/
+    chrome --headless --disable-gpu --dump-dom https://www.chromestatus.com/
```

Comment:
It looks like you need `--disable-gpu` on every command, so you should change the note on line 69